crates/noirc_evaluator/src/ssa_refactor/ir/dfg.rs

```diff
@@ -91,6 +94,15 @@ impl DataFlowGraph {
         new_block
     }
 
+    /// Brillig compilation artefact
+    pub(crate) fn obj(&self) -> &Artefact {
+        &self.obj
+    }
+
+    pub(crate) fn brillig_gen(&mut self, obj: Artefact) {
+        self.obj = obj;
+    }
```

Comment:
Per the last comment I'd like to remove the `obj` field entirely, but its worth mentioning we should have clearer names as well, avoiding general words like "obj." `Artefact` as well is rather general. Perhaps `Artefact` can be `Brillig` and we would have a corresponding `self.brillig: Brillig` field for objects with Brillig code.

Reply:
Done